From 559605ed38c7115d08f60ef914b6c5614095eaad Mon Sep 17 00:00:00 2001
From: Enrico Joerns <ejo@pengutronix.de>
Date: Thu, 8 Mar 2018 12:25:25 +0100
Subject: [PATCH 39/44] src/update_handler: support vfat to vfat

This adds support for writing a .vfat image (which should be a valid
extension for a vfat image) to a vfat slot by using the img_to_fs_handler.

Signed-off-by: Enrico Joerns <ejo@pengutronix.de>
---
 src/update_handler.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/update_handler.c b/src/update_handler.c
index 688123a..22a10e8 100644
--- a/src/update_handler.c
+++ b/src/update_handler.c
@@ -1105,6 +1105,7 @@ RaucUpdatePair updatepairs[] = {
 	{"*.ext4", "ext4", img_to_fs_handler},
 	{"*.ext4", "raw", img_to_raw_handler},
 	{"*.vfat", "raw", img_to_raw_handler},
+	{"*.vfat", "vfat", img_to_fs_handler},
 	{"*.tar*", "ext4", archive_to_ext4_handler},
 	{"*.catar", "ext4", archive_to_ext4_handler},
 	{"*.tar*", "ubifs", archive_to_ubifs_handler},
-- 
2.14.1

